GitUI/CommandsDialogs/WorktreeDialog/FormCreateWorktree.cs

```diff
@@ -109,7 +109,11 @@ private void CreateWorktree()
             {
                 "add",
                 WorktreeDirectory.Quote(),
-                { radioButtonCreateNewBranch.Checked, $"-b {textBoxNewBranchName.Text}", ((GitRef)comboBoxBranches.SelectedItem).Name }
+                {
+                    radioButtonCreateNewBranch.Checked,
+                    $"-b {textBoxNewBranchName.Text}",
+                    comboBoxBranches.SelectedItem != null ? ((GitRef)comboBoxBranches.SelectedItem).Name : null
```

Comment:
Just a suggestion to have this shorter: `((GitRef)comboBoxBranches.SelectedItem)?.Name`